server/v2/cometbft/commands.go

```diff
@@ -28,35 +28,40 @@ import (
 	"github.com/cosmos/cosmos-sdk/version"
 )
 
-func (s *CometBFTServer[T]) rpcClient(cmd *cobra.Command) (rpc.CometRPC, error) {
-	if s.config.AppTomlConfig.Standalone {
+func rpcClient(cmd *cobra.Command) (rpc.CometRPC, error) {
+	v := client.GetViperFromCmd(cmd)
```

Comment:
when doing queries it shouldn't need to read the app.toml actually right, but the client.toml

Reply:
Yeah flag node will get if from client.toml, so that's great. Not sure what a standalone check means for query tbh.